<html>
<head>
<link rel="shortcut icon" href="images/iob.ico" type="image/x-icon" />
<title>Institute of Bioinformatics</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<meta name="description" content="The Institute of Bioinformatics (IOB) established by Dr. Akhilesh Pandey, is a non-profit academic research organization located at International Tech Park in Bangalore, India. IOB is recognized by the Department of Scientific and Industrial Research, Government of India." /> 
<meta name="keywords" content="bioinformatics, india, bangalore, database, institute, proteomics, mass spectrometry, mass spectrometer, LTQ-Orbitrap, proteomics bangalore, india, genomics, protein reference database, oncomine, HPRD, akhilesh, akhilesh pandey, hprd, plasma proteome database, netpath, pathways, signaling pathways, pathbuilder, biobuilder, proteinpedia, human proteinpedia, SVM, RAPID, primary Immunodeficiency disease, cancer, biomarkers, biomarker, gstric cancer, pancreatic cancer, esophageal cancer, ESCC, X chromosome, XLMR, microarray, CGH array, neurological disorders, epilepsy, rabies, CNV, protein tyrosine phosphatases, gene expression arrays, tiling arrays, tissue microarrays, cell pellet arrays, breast cancer database, BCD, oncomine, phosphoproteomics, SILAC, iTRAQ, Glycoproteomics, proteogenomics, secretome, neuroproteomics, toxicoproteomics, plant proteomics, ethanoproteomics, serum proteomics, quantitative proteomics, HPLC, OFFGEL, nano-LC, ETD, Electron Transfer Dissociation, mycobacterium tuberculosis, research, proteomcis research, biomarker discovery, arsenic, Ph.D., manipal, kuvempu, pondicherry, amrita, rajiv gandhi university of health science, KIDWAI, NIMHANS, Johns Hopkins University, JHMI, RIKEN, RCAI, University of Toronto, IGIB, IICB, NIMR, MSKCC, sloan-kettering, University of Southern Denmark, HUPO, comparative proteomics, embryonic stem cells, anopheles, iob, comparative genomics, est">
<link rel="stylesheet" href="css/style.css" />
<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.5.2/jquery.min.js"></script>
<script type="text/javascript" src="jquery.aw-showcase.js"></script>
<script type="text/javascript">

$(document).ready(function()
{
	$("#showcase").awShowcase(
	{
	content_width:			540,
		content_height:			236,
		fit_to_parent:			false,
		auto:					true,
		interval:				3000,
		continuous:				false,
		loading:				true,
		tooltip_width:			200,
		tooltip_icon_width:		32,
		tooltip_icon_height:	32,
		tooltip_offsetx:		18,
		tooltip_offsety:		0,
		arrows:					true,
		buttons:				true,
		btn_numbers:			true,
		keybord_keys:			true,
		mousetrace:				false, /* Trace x and y coordinates for the mouse */
		pauseonover:			true,
		stoponclick:			false,
		transition:				'hslide', /* hslide/vslide/fade */
		transition_delay:		0,
		transition_speed:		500,
		show_caption:			'onload', /* onload/onhover/show */
		thumbnails:				false,
		thumbnails_position:	'outside-last', /* outside-last/outside-first/inside-last/inside-first */
		thumbnails_direction:	'vertical', /* vertical/horizontal */
		thumbnails_slidex:		1, /* 0 = auto / 1 = slide one thumbnail / 2 = slide two thumbnails / etc. */
		dynamic_height:			false, /* For dynamic height to work in webkit you need to set the width and height of images in the source. Usually works to only set the dimension of the first slide in the showcase. */
		speed_change:			true, /* Set to true to prevent users from swithing more then one slide at once. */
		viewline:				false, /* If set to true content_width, thumbnails, transition and dynamic_height will be disabled. As for dynamic height you need to set the width and height of images in the source. */
		custom_function:		null /* Define a custom function that runs on content change */
	});
});

</script>
<script language="JavaScript">
<!--
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_openBrWindow(theURL,winName,features) { //v2.0
  window.open(theURL,winName,features);
}
//-->
</script>

<link href="style.css" rel="stylesheet" type="text/css"></head>
<body bgcolor="#E6E6E6" text="#000000" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0" onLoad="MM_preloadImages('images/collagiate/iob_roll.gif','images/buttons/csa_roll.gif','images/buttons/director_roll.gif','images/buttons/board_roll.gif','images/buttons/links_roll.gif','images/collagiate/team_roll.gif')">
<table width="801" border="0" align="center" cellpadding="0" cellspacing="0">

  <tr> 
    <td width="801"  bgcolor="#FFFFFF" align="center"><?php require("header.htm");?> &nbsp;</td>
  </tr> 
   <tr>
  <td>
  
  <table bgcolor="#FFFFFF" width="804">
  <tr><td width="129" height="42" valign="top"><?php include('menu.html');?> </td>
    <td width="663" valign="top" align="left"><fieldset>
    <table width="100%" border="0" cellspacing="0" cellpadding="0">
      <!--<tr>
                <td></td>
                <td width="95%" colspan="2">&nbsp;</td>
                <td>&nbsp;</td>
              </tr>
              
              <tr> 
                <td colspan="3"  class="header_title" style="border-bottom:#387A9A thick double; font:Geneva, Arial, Helvetica, sans-serif; font-size:13px"><strong>Visitors' Gallery</strong></td>
                <td width="5%">&nbsp;</td>
              </tr>-->
      <tr>
        <tr>
    <td colspan="5" bgcolor="#FFFFFF"  style="border-bottom:#0066FF medium groove; font:Geneva, Arial, Helvetica, sans-serif; font-size:14px" align="center"><strong>               BIOINFORMATICS &#8250; RESEARCH
        AREAS </strong></td>
    
  </tr>
      <tr>
        <td colspan="3" align="left">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="3" align="left" valign="top"><table width="100%" border="0" cellspacing="0" cellpadding="0">
          <!--  <tr>
                <td></td>
                <td width="95%" colspan="2">&nbsp;</td>
                <td>&nbsp;</td>
              </tr>
              
              <tr> 
                <td colspan="3"  class="header_title" style="border-bottom:#387A9A thick double; font:Geneva, Arial, Helvetica, sans-serif; font-size:13px"><strong>Bioinformatics &#8250; Research Areas </strong></td>
                <td width="5%">&nbsp;</td>
              </tr>-->
          <tr>
            <td colspan="3" align="center"><a href="http://www.netpath.org/netslim/EGFR1_full.html" target="_blank" class="bold">EGFR1
                Signaling Pathway</a><br>
              <a href="http://www.netpath.org/netslim/EGFR1_full.html" target="_blank"><img src="images/EGFR1.jpg" width="580" height="125" border="0"  style="padding: 3; border: 1px solid #cccccc;"></a></td>
          </tr>
          <tr>
            <td colspan="3" align="left">&nbsp;</td>
          </tr>
          <tr>
            <td colspan="3" align="left">
              <table   border="0" cellpadding="0" cellspacing="0">
                <tr>
                  <td class="footer"  align="center" 
width="122" height="16" ><a href="bioinformatics.php"><img src="images/ResearchAreas.gif" border="0"></a></td>
                  <td class="footer"  align="center" ><a href="infrastructure.php"><img src="images/Infrastructure_black.gif" border="0"></a></td>
                </tr>
              </table>
              <table class="double5">
                <td colspan="4">&nbsp; </td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"> <img src="images/bcd_logo.jpg" width="175" height="25"></td>
                  <td><p align="justify"><a href="http://www.breastcancerdatabase.org" target="_blank" class="header">BREAST
                        CANCER DATABASE</a></p>
                      <p align="justify"> A database which contains all the &#8220;Known
                        molecular alterations related to Breast Cancer&#8221;.
                        This database comprises of information related to known
                        molecular alterations at the chromosome, mRNA and protein
                        levels. The database will also comprise all the SAGE
                        analysis and Microarray data relevant to breast cancer.
                        The data includes information on regulation (up/down)
                        of all the reported genes and its proteins related to
                        breast cancer. This database is linked to the HPRD for
                        manually curated protein annotations, ONCOMINE, Geneontology,
                        Unigene, and Genecard, where the user can have a comparative
                        account of cancerous and normal state for a particular
                        gene of their interest.</p>
		<p align="justify">
			<a href="http://www.breastcancerdatabase.org" title="Breast Cancer Database" target="_blank">
				<img src="images/more.gif" width="39" height="15" border="0">
			</a>
		</p>
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><a href="http://netpath.org/"><img src="images/netpath.gif" border="0"></a></td>
                  <td valign="top" class="header"><p align="justify"><a name="pathways"></a><a href="http://netpath.org/"><span class="header">NETPATH</span></a></p>
                      <p align="justify">'NetPath' is a curated resource of signal
                        transduction pathways in humans. It is a collaboration
                        between the PandeyLab at Johns Hopkins University and
                        the Institute of Bioinformatics. At this time, 10 cancer
                        and 10 immune signaling pathways are available. For example
                        Transforming growth factor-beta (TGF-beta) is a multifunctional
                        growth factor with a wide range of biological activities
                        such as cell differentiation, cell proliferation, apoptosis
                        and tumor suppression. TGF-beta signalling involves a
                        direct pathway from the cell surface receptors to the
                        nucleus. TGF- beta, binding to its receptors initiates
                        the degradation of several key components of its signaling
                        pathway. The degradation of these components, including
                        both positive and negative transducers, is mediated by
                        the ubiquitinated proteasome system. Recent cellular,
                        biochemical, and structural studies have shown significant
                        insight into the mechanisms of the activation of TGF-beta
                        receptors through ligand binding, the activation of Smad
                        proteins through phosphorylation, the transcriptional
                        regulation of target gene expression, and the control
                        of Smad protein activity and degradation.</p>
			<p align="justify">
				<a href="http://netpath.org/" title="NetPath" target="_blank">
					<img src="images/more.gif" width="39" height="15" border="0" >
				</a>
			</p>
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><a href="http://pathbuilder.sourceforge.net/" target="_blank"><img src="images/pathbuilder.gif" border="1"></a></td>
                  <td><p align="justify"><a href="http://pathbuilder.sourceforge.net/" class="header">PathBuilder</a></p>
                      <p align="justify">PathBuilder is an open source software
                        to annotate biological information pertaining to signaling
                        pathways and, with minimal additional effort, to create
                        web-based pathway resources. PathBuilder enables annotation
                        of molecular events including protein-protein interactions,
                        enzyme-substrate relationships and protein translocation
                        events via manual or automatic methods. The features
                        of PathBuilder include automatic validation of data formats,
                        built-in modules for visualizing pathways, automated
                        import of data from other pathway resources, export of
                        data in several standard data exchange formats and an
                        application programming interface for retrieving pathway
                        datasets.</p>
                      <p align="justify">
			<a href="http://pathbuilder.sourceforge.net/" target="_blank">
				<img src="images/more.gif" width="39" height="15" border="0">
			</a>
		      </p>
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><a href="http://rapid.rcai.riken.jp/"><img src="images/rapid_logo.gif" border="0"></a></td>
                  <td valign="top" class="header"><p align="justify"><a name="rapid"></a><a href="http://rapid.rcai.riken.jp/"><span class="header">Resource
                          of Asian Primary Immunodeficiency Diseases</span></a></p>
                      <p align="justify">To build a PID informational platform
                        and also as a part of action to initiate a network of
                        PID research in Asia, we have constructed a web-based
                        compendium of molecular alterations in PID, named Resource
                        of Asian Primary Immunodeficiency Diseases (RAPID) in
                        collaboration with RIKEN, Japan. It hosts information
                        on sequence variations and expression at the mRNA and
                        protein levels of all genes reported to be involved in
                        PID patients. The main objective of this database is
                        to provide detailed information pertaining to genes and
                        proteins involved in primary immunodeficiency diseases
                        along with other relevant information about protein-protein
                        interactions, mouse studies and microarray gene-expression
                        profiles in various organs and cells of the immune system.
                        RAPID also contains list of candidate PID genes predicted
                        using support vector machine learning approach.Thus,
                        information contained in this database should help physicians
                        and other biomedical investigators to further investigate
                        the role of these molecules in PID.</p>
                      <p align="justify"><a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('http://rapid.rcai.riken.jp/','RAPID','scrollbars=yes,width=600,height=700')"></a></p>
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td height="12" colspan="4" align="center">&nbsp;</td>
                <tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><a href="http://humanproteinpedia.org/"><img src="images/proteinpedia.gif" border="0"></a></td>
                  <td valign="top" class="header"><p align="justify"><a name="pathways"></a><a href="http://humanproteinpedia.org/"><span class="header">HUMAN
                          PROTEINPEDIA</span></a></p>
                      <p align="justify">Human Proteinpedia is a community portal
                        for sharing and integration of human protein data. It
                        allows research laboratories to contribute and maintain
                        protein annotations. Human Protein Reference Database
                        (HPRD) integrates data, that is deposited in Human Proteinpedia
                        along with the existing literature curated information
                        in the context of an individual protein. All the public
                        data contributed to Human Proteinpedia can be queried,
                        viewed and downloaded.</p>
                      <p align="justify"><a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('http://humanproteinpedia.org/','proteinpedia','scrollbars=yes,width=600,height=700')"></a></p>
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td height="12" colspan="4" align="center">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td width="31%" align="center"><a href="http://www.hprd.org" target="_blank"><img src="images/hprd_logo.gif" width="177" height="89" border="0"></a></td>
                  <td width="64%"><p align="justify"><a href="http://www.hprd.org/" target="_blank" class="header">HUMAN
                        PROTEIN REFERENCE DATABASE (HPRD)</a></p>
                      <p align="justify">First of its kind, a database hosting
                        over 25000 human proteins and disease genes. The main
                        objective of this database is to make available all related
                        fields of a protein including Interactions, Post Translational
                        Modifications, Substrates and other information so as
                        to help the scientists working in the areas of Proteomics.
                        This makes HPRD a reliable source for protein data. HPRD
                        is the first ever database to implement the standardization
                        protocol put forward by the Proteomics Standards Initiative
                        for molecular interactions (PSI-MI). The annotations
                        are manually done by scientists working at IOB and each
                        annotation undergoes multiple levels of reviews before
                        being made publicly available, to ensure the quality.
                        The database host annotations of all the known human
                        proteins and update them on an ongoing basis.</p>
                      <p align="justify"><a href="http://www.hprd.org" target="_blank"><img src="images/more.gif" width="39" height="15" border="0"></a></p>
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td width="31%" align="center"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15099404" target="_blank"><img src="images/biobuilder_logo.jpg" width="177" height="57" border="0"></a></td>
                  <td width="64%"><p align="justify"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15099404" target="_blank" class="header">Bio-Builder</a></p>
                      <p align="justify">A content management system, designed
                        for easy modifications and management of HPRD, using
                        web interface. The administration of annotation process
                        and the multi-level review carried out round the globe
                        is made easy by the implementation of this tool. This
                        user friendly tool is created using Python and Zope.</p>
                      <p align="justify"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15099404" target="_blank"><img src="images/more.gif" width="39" height="15" border="0"></a></p>
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr> <!--
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><img src="images/genomebrowser_logo.jpg" width="177" height="52" border="0"></td>
                  <td><p align="justify"><a href="" class="header">GENOME BROWSER</a></p>
                      <p align="justify">An application which would comprise
                        the basic sequence data along with the higher-level,
                        location-based annotations on the sequence: These annotations
                        would relate the sequence data to various genomic aspects
                        like SNPs and expression profiling data such as that
                        derived from DNA microarray and SAGE studies and also
                        mass spectrometry derived data in order to understand
                        various post translational modifications. This would
                        add value to the analysis of the genomic sequence, as
                        it would provide several different contexts in which
                        to interpret the genomic/transcriptomic/proteomic data.
                        With this application, when exploring a genomic region,
                        biologists would be able to interact with the interface
                        in a richer fashion than is currently possible using
                        simple, hyperlinked images.</p>
                      <p align="justify"><a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('/projects/genomebrowser.htm','genomebrowser','scrollbars=yes,width=500,height=600')"></a></p>
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr> -->
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td width="31%" align="center"><a href="http://plasmaproteomedatabase.org/" target="_blank"><img src="images/ppd.gif" border="1"></a></td>
                  <td width="64%"><p align="justify"><a href="http://plasmaproteomedatabase.org/" target="_blank" class="header">PLASMA
                        PROTEOME DATABASE (PPD)</a></p>
                      <p align="justify">The Plasma Proteome Database, the first
                        of its kind ensures a comprehensive resource for all
                        human plasma proteins along with their isoforms. The
                        database includes information pertaining to isoform specific
                        expression, disease, localization, post translational
                        modification and single nucleotide polymorphism. The
                        information provided in this database is through manual
                        annotation done by exhaustive literature research.</p>
                      <p align="justify"><a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('http://plasmaproteomedatabase.org/','ppd','width=600,height=700')"></a></p>
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><img src="images/tagmapper_logo.jpg" width="177" height="52" border="0"></td>
                  <td><p align="justify"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16112519" target="_blank"><b>TAGmapper</b><a/></p>
                      <p align="justify">Is a comprehensive tool used to perform
                        tag-to-gene mapping. The 10 base pair sequences of each
                        of the SAGE tags, which are generated experimentally
                        in the lab, are submitted to the tool as input. The output
                        results in mapping the SAGE tags to their respective
                        genes by performing an extensive search across the &#8216;dbEST&#8217; and &#8216;non
                        redundant&#8217; database.</p>
                      <p align="justify"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16112519" target="_blank"><img src="images/more.gif" width="39" height="15" border="0"></a></p>
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><a href="#" target="_blank"><img src="images/cg_logo.jpg" width="177" height="53" border="0"></a></td>
                  <td><p align="justify">
			<!-- <a href="#http://203.200.58.139/CG_Proj" target="_blank" class="header"> -->
			<b>ANALYSIS OF HUMAN 'X' CHROMOSOME</b></p>
                      <p align="justify">Involves a careful and comprehensive
                        analysis of Human 'X' chromosome. Using comparative genomic
                        approach, we have identified novel protein coding regions;
                        we have performed an extensive pseudogene analysis of
                        the 'X' chromosome and have documented alternative splicing
                        events. In order to help the scientific community working
                        on X-Linked Mental Retardation, the domain is linked
                        to HPRD where the user can view the annotations of all
                        the genes on 'X'.</p>
                      <p align="justify"><img src="images/more.gif" width="39" height="15" border="0"></p>
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><img src="images/ptpr_logo.jpg" width="175" height="40"></td>
                  <td><p align="justify">
			<!-- <a href="http://ptpr.ibioinformatics.org" class="header"> -->
			<b>ANALYSIS OF HUMAN PROTEIN TYROSINE PHOSPHATASES FOR ALTERNATIVE
                        SPLICING</b></p>
                      <p align="justify"> A thorough analysis of the protein
                        tyrosine phosphatases encoded by the human genome using
                        computational biology methods. Primary aim of the study
                        is to identify novel tyrosine phosphatases and novel
                        transcript variants for all the known protein tyrosine
                        phosphatases. The findings are then experimentally validated
                        by experts using techniques such as RT-PCRs and Northern
                        blots.</p>
                      <p align="justify"></p>
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr>

	<tr>
		<td valign="top">&nbsp;</td>
		<td align="center">
			<a href="http://www.oncomine.org" target="_blank">
				<img src="images/oncomine_logo.jpg" width="175" height="27" border="0">
			</a>
		</td>
		<td>
			<p align="justify">
				<a href="http://www.oncomine.org/" target="_blank" class="header"> ONCOMINE</a>
			</p>
			<p align="justify">IOB, in collaboration with Chinnaiyan Lab, has created a microarray database whose goal is
                        to curate publicly available cancer microarray studies and provide data mining tools to generate biologically
                        relevant information in a user friendly manner. Links to various bioinformatics resources have been implemented
                        including Unigene, Swissprot, Biocarta, HPRD, and KEGG, among others. IOB was involved in implementing the technical
                        aspect for the database and Chinnaiyan Lab provided the normalized expression data. This database has been published
                        in January/February issue of Neoplasia.</p>
			<p align="justify">
				<a href="http://www.oncomine.org" target="_blank">
					<img src="images/more.gif" width="39" height="15" border="0">
				</a>
			</p>
 		</td>
		<td>&nbsp;</td>
	</tr>

	<tr>
		<td valign="top">&nbsp;</td>
		<td align="center">
			<a href="" target="_blank">
				<img src="images/incredb.png" width="175" height="27" border="0">
			</a>
		</td>
		<td>
			<p align="justify">
				<a href="http://www.incredb.org/" target="_blank" class="header">India Cancer Research Database</a>
			</p>
			<p align="justify">This database provides details of scientists and physicians involved in cancer research in India along with the information about their areas of expertise, research publications and funded grants. The main goal of the database was to foster collaborations among researchers and to provide a snapshot of ongoing research initiatives and activities in India. </p>
			<p align="justify">
				<a href="http://www.incredb.org/" target="_blank">
					<img src="images/more.gif" width="39" height="15" border="0">
				</a>
			</p>
 		</td>
		<td>&nbsp;</td>
	</tr>


<!--Template for Next Time
	<tr>
		<td valign="top">&nbsp;</td>
		<td align="center">
			<a href="http://www.oncomine.org" target="_blank">
				<img src="images/oncomine_logo.jpg" width="175" height="27" border="0">
			</a>
		</td>
		<td>
			<p align="justify">
				<a href="http://www.oncomine.org/" target="_blank" class="header"> ONCOMINE</a>
			</p>
			<p align="justify">CONTENTs</p>
			<p align="justify">
				<a href="http://www.oncomine.org" target="_blank">
					<img src="images/more.gif" width="39" height="15" border="0">
				</a>
			</p>
 		</td>
		<td>&nbsp;</td>
	</tr>
 --.
                <tr>
                  <td colspan="4" align="right"><a href="#top">TOP</a></td>
                </tr>
              </table>
            </td>
          </tr>
          <tr>
            <td colspan="3" align="left">&nbsp;</td>
          </tr>
        </table></td>
      </tr>
      <tr>
        <td colspan="3" align="left">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="3" align="left">&nbsp;</td>
      </tr>
    </table>
    
	
	</fieldset></td>
  </tr>
  
  
  </table>
  
  
  
  
  </td></tr> 
    </table>


<br>

<br>
</body>
</html>
