<html>
<head>
<link rel="shortcut icon" href="images/iob.ico" type="image/x-icon" />
<title>Institute of Bioinformatics</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<meta name="description" content="The Institute of Bioinformatics (IOB) established by Dr. Akhilesh Pandey, is a non-profit academic research organization located at International Tech Park in Bangalore, India. IOB is recognized by the Department of Scientific and Industrial Research, Government of India." /> 
<meta name="keywords" content="bioinformatics, india, bangalore, database, institute, proteomics, mass spectrometry, mass spectrometer, LTQ-Orbitrap, proteomics bangalore, india, genomics, protein reference database, oncomine, HPRD, akhilesh, akhilesh pandey, hprd, plasma proteome database, netpath, pathways, signaling pathways, pathbuilder, biobuilder, proteinpedia, human proteinpedia, SVM, RAPID, primary Immunodeficiency disease, cancer, biomarkers, biomarker, gstric cancer, pancreatic cancer, esophageal cancer, ESCC, X chromosome, XLMR, microarray, CGH array, neurological disorders, epilepsy, rabies, CNV, protein tyrosine phosphatases, gene expression arrays, tiling arrays, tissue microarrays, cell pellet arrays, breast cancer database, BCD, oncomine, phosphoproteomics, SILAC, iTRAQ, Glycoproteomics, proteogenomics, secretome, neuroproteomics, toxicoproteomics, plant proteomics, ethanoproteomics, serum proteomics, quantitative proteomics, HPLC, OFFGEL, nano-LC, ETD, Electron Transfer Dissociation, mycobacterium tuberculosis, research, proteomcis research, biomarker discovery, arsenic, Ph.D., manipal, kuvempu, pondicherry, amrita, rajiv gandhi university of health science, KIDWAI, NIMHANS, Johns Hopkins University, JHMI, RIKEN, RCAI, University of Toronto, IGIB, IICB, NIMR, MSKCC, sloan-kettering, University of Southern Denmark, HUPO, comparative proteomics, embryonic stem cells, anopheles, iob, comparative genomics, est">
<link rel="stylesheet" href="css/style.css" />
<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.5.2/jquery.min.js"></script>
<script type="text/javascript" src="jquery.aw-showcase.js"></script>
<script type="text/javascript">

$(document).ready(function()
{
	$("#showcase").awShowcase(
	{
	content_width:			540,
		content_height:			236,
		fit_to_parent:			false,
		auto:					true,
		interval:				3000,
		continuous:				false,
		loading:				true,
		tooltip_width:			200,
		tooltip_icon_width:		32,
		tooltip_icon_height:	32,
		tooltip_offsetx:		18,
		tooltip_offsety:		0,
		arrows:					true,
		buttons:				true,
		btn_numbers:			true,
		keybord_keys:			true,
		mousetrace:				false, /* Trace x and y coordinates for the mouse */
		pauseonover:			true,
		stoponclick:			false,
		transition:				'hslide', /* hslide/vslide/fade */
		transition_delay:		0,
		transition_speed:		500,
		show_caption:			'onload', /* onload/onhover/show */
		thumbnails:				false,
		thumbnails_position:	'outside-last', /* outside-last/outside-first/inside-last/inside-first */
		thumbnails_direction:	'vertical', /* vertical/horizontal */
		thumbnails_slidex:		1, /* 0 = auto / 1 = slide one thumbnail / 2 = slide two thumbnails / etc. */
		dynamic_height:			false, /* For dynamic height to work in webkit you need to set the width and height of images in the source. Usually works to only set the dimension of the first slide in the showcase. */
		speed_change:			true, /* Set to true to prevent users from swithing more then one slide at once. */
		viewline:				false, /* If set to true content_width, thumbnails, transition and dynamic_height will be disabled. As for dynamic height you need to set the width and height of images in the source. */
		custom_function:		null /* Define a custom function that runs on content change */
	});
});

</script>
<script language="JavaScript">
<!--

function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}
//-->
</script>

<link href="style.css" rel="stylesheet" type="text/css"></head>
<body bgcolor="#E6E6E6" text="#000000" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0" onLoad="MM_preloadImages('images/collagiate/iob_roll.gif','images/buttons/csa_roll.gif','images/buttons/director_roll.gif','images/buttons/board_roll.gif','images/buttons/links_roll.gif','images/collagiate/team_roll.gif')">
<table width="801" border="0" align="center" cellpadding="0" cellspacing="0">

  <tr> 
    <td width="801"  bgcolor="#FFFFFF" align="center"><?php require("header.htm");?> &nbsp;</td>
  </tr> 
   <tr>
  <td>
  
  <table bgcolor="#FFFFFF" width="804">
  <tr><td width="129" height="42" valign="top"><?php include('menu.html');?> </td>
    <td width="663" valign="top" align="left"><fieldset>
    <table width="100%" border="0" cellspacing="0" cellpadding="0">
      <!--<tr>
                <td></td>
                <td width="95%" colspan="2">&nbsp;</td>
                <td>&nbsp;</td>
              </tr>
              
              <tr> 
                <td colspan="3"  class="header_title" style="border-bottom:#387A9A thick double; font:Geneva, Arial, Helvetica, sans-serif; font-size:13px"><strong>Visitors' Gallery</strong></td>
                <td width="5%">&nbsp;</td>
              </tr>-->
      <tr>
        <tr>
    <td colspan="5" bgcolor="#FFFFFF"  style="border-bottom:#0066FF medium groove; font:Geneva, Arial, Helvetica, sans-serif; font-size:14px" align="center"><strong>               PROTEOMICS  &#8250; RESEARCH
        AREAS </strong></td>
    
  </tr>
      <tr>
        <td colspan="3" align="left">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="3" align="left" valign="top"><table width="100%" border="0" cellspacing="0" cellpadding="0">
          <!--  <tr>
                <td></td>
                <td width="95%" colspan="2">&nbsp;</td>
                <td>&nbsp;</td>
              </tr>
              
              <tr> 
                <td colspan="3"  class="header_title" style="border-bottom:#387A9A thick double; font:Geneva, Arial, Helvetica, sans-serif; font-size:13px"><strong>Bioinformatics &#8250; Research Areas </strong></td>
                <td width="5%">&nbsp;</td>
              </tr>-->
          <tr>
            <td colspan="3" align="center"><table width="100%" border="0">
              <tr>
                <td valign="middle" align="center"><img src="images/iob_new_images/proteomicsFandM.jpg" style="padding: 3; border: 1px solid #cccccc;"></td>
                <td valign="middle" align="center"><img src="images/proteomics_lab_2.gif" style="padding: 3; border: 1px solid #cccccc;"></td>
              </tr>
            </table>
              </td>
          </tr>
          <tr>
            <td colspan="3" align="left">&nbsp;</td>
          </tr>
          <tr>
            <td colspan="3" align="left" valign="top">
              <table   border="0" cellpadding="0" cellspacing="0">
                <tr>
                  <td class="footer"  align="center" 
width="122" height="16" ><a href="proteomics.php"><img src="images/ResearchAreas.gif" border="0"></a></td>
                  <td class="footer"  align="center" ><a href="proteomics_facility.php"><img src="images/Infrastructure_black.gif" border="0"></a></td>
                </tr>
              </table>
              <table class="double5">
                <td colspan="4">&nbsp; </td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><img src="images/qproteomics.gif" style="padding: 4px; border: 1px solid #000000;" width="170" height="119" border="0"></td>
                  <td><p><span class="header">Quantitative proteomics </span></p>
                      <p align="justify">We use quantitative proteomics approaches
                        for the identification of candidate biomarkers in human
                        diseases and infectious microorganisms. We have pioneered
                        the use of in vivo labeling strategies such as SILAC
                        (stable isotope labeling with amino acids in cell culture),
                        a technology that is currently being used by numerous
                        groups around the world.&nbsp; We are also experts in
                        other quantitative labeling methodologies such as 18O
                        labeling and iTRAQ &ndash;in fact, we were early adopters
                        of the 8-plex labeling reagents, which will be the mainstay
                        of many of the studies to be carried out under this initiative.&nbsp; We
                        characterized proteome of various cancers including breast,
                        pancreatic, hepatocellular, lung and billiary cancers.
                        We are adopting same technologies to identify molecular
                        profiling of esophageal squamous cell, gastric and gall
                        bladder carcinoma.&nbsp; Similarly, we are analyzing
                        proteome of other diseases such as chronic meningitis,
                        neurological stroke, temporal lobe epilepsy and rabies.</p>
                      <!--<p><a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('/projects/bcd.htm','bcd','scrollbars=yes,width=500,height=600')"></a></p>-->
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><img src="images/phosphoproteomics.gif" style="padding: 4px; border: 1px solid #000000;" width="177" height="92" border="0"></td>
                  <td valign="top" class="header"><p><a name="pathways"></a><span class="header">Phosphoproteomics</span></p>
                      <p align="justify">We are investigating phosphoproteomic
                        profiling associated with various cancers and other diseases.
                        We are also profiling phosphotyrosines in various human
                        tissues, cancers, other diseases and several microbes
                        of clinical and industrial importance. Hyperphosphorylated
                        and hypophosphorylated molecules associated with a particular
                        disease, virulence or drug resistance collectively may
                        provide information about activation or down-regulation
                        of associated signaling pathways or molecular mechanisms.
                        We combine the advantage of the SILAC or iTRAQ method
                        to obtain quantitative data with TiO2 and anti- phosphopeptide
                        antibody based enrichment of phosphopeptides.&nbsp; We
                        were also the first to apply a novel mass spectrometric
                        technology, Electron Transfer Dissociation (ETD), for
                        global phosphoproteomic analysis. ETD is a preferred
                        method of fragmentation in phosphoproteomic studies as
                        it preserves phosphate moieties on serine and threonine
                        residues that are otherwise quite labile in conventional
                        collision induced dissociation.</p>
                      <!--<p><a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('/projects/netpath.htm','netpath','width=600,height=700,scrollbars=yes')"></a></p>-->
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><img src="images/proteomics.jpg" style="padding: 4px; border: 1px solid #000000;" width="190" height="150" border="0"></td>
                  <td><p><span class="header">Proteomics of Body Fluids</span></p>
                      <p align="justify">The main aim of this research is to generate and quantitatively identify the complete proteomic 		profiles of every protein expressed in various human body fluids using our state-of-the-art and highly sensitive mass spectroscopy-based proteomic techniques. We summarize some of the strategies currently being used for global identification and quantification of body fluid proteins with applications for a wide variety of human diseases. Our findings reveal novel protein species in both healthy and disease states that were not previously identified. While it is reasonable to expect that some of the same proteins are found in multiple body fluids, the primary objective of this research will be to identify a definable subset of proteins that were unique to or diagnostic for a single body fluid. The study will evaluate the entire complement of proteins in each proteome for potential utility as bodily fluid specific biomarkers. Consequently we hope that some of these biomarkers can be further characterized for use in highly sensitive diagnostic tests or for evaluating therapeutic response. With the advances made in proteomics technologies, the impact of this analysis in the search for clinically relevant disease biomarkers would be realized in the future. </p>
                      <p><a href="proteomics_body_fluids.php"><img src="images/more.gif" width="39" height="15" border="0" ></a></p>
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><img src="images/glycoproteomics.gif" style="padding: 4px; border: 1px solid #000000;" width="177" height="56" border="0"></td>
                  <td valign="top" class="header"><p><span class="header">Glycoproteomics</span></p>
                      <p align="justify">We are using glycoproteome enrichment
                        techniques in order to detect usually less abundant glycosylated
                        proteins that are more relevant from the point of view
                        of biomarker development<strong>.</strong> We have substantial
                        experience in enrichment of glycoproteins using lectin
                        affinity chromatography. The enrichment is carried out
                        using a mixture of three lectins - concanavalin A, wheat
                        germ agglutinin and jacalin- that are immobilized on
                        a matrix.&nbsp; Use of 3 lectins provides a broader coverage
                        of glycoproteins because of their broader binding profile
                        and can provide up to 1,000 fold enrichment of molecules
                        in a single step. For identification of N-linked glycopeptides,
                        the extracted tryptic peptides are treated with PNGaseF.&nbsp; This
                        digestion step removes the N-linked glycan and is recognized
                        by a deamidation at the glycan linked asparagine during
                        cleavage. This converts asparagine to aspartic acid increasing
                        the mass by 1Da that could be very easily identified
                        by mass spectrometry.</p>
                      <!--<p><a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('http://rapid.rcai.riken.jp/','RAPID','width=600,height=700,scrollbars=yes')"></a></p>-->
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td height="12" colspan="4" align="center">&nbsp;</td>
                <tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><img src="images/cellpellet.gif" style="padding: 4px; border: 1px solid #000000;" width="177" height="133" border="0"></td>
                  <td valign="top" class="header"><p><span class="header">Cell
                        line secretome</span></p>
                      <p align="justify">The secretome of cells and tissues reflects
                        a wide range of pathological conditions and represents
                        as a good source of biomarkers. The identification of
                        secreted proteins is usually hard as it is hardly accessible
                        by direct proteome analysis because these proteins are
                        often masked by high amounts of proteins not secreted
                        by the cells under investigation. Therefore, we study
                        the secretome from patient derived cell lines as that
                        can give a better idea of the secretome abundance of
                        protein in tumor as oppose to normal. We have successfully
                        used SILAC technology and cell line secretome approach
                        to investigate secreted biomarkers of pancreatic and
                        esophageal cancers. </p>
                      <!--<p><a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('http://humanproteinpedia.org/','proteinpedia','width=600,height=700,scrollbars=yes')"></a></p>-->
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td height="12" colspan="4" align="center">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td width="31%" align="center"><img src="images/proteogenomics.gif" style="padding: 4px; border: 1px solid #000000;" width="200" height="52" border="0"></td>
                  <td width="64%"><p><span class="header">Proteogenomics</span></p>
                      <p align="justify">Proteogenomics is the annotation of
                        genomes using proteomics data. Mass spectrometry derived
                        data can be searched against genome which will provide
                        the evidence for protein coding potential.&nbsp; Successful
                        outcome of such analyses include identification of novel
                        genes, novel exons, novel initiation codons, exon extensions,
                        intronic genes and cSNPs. We are carrying out proteogenomic
                        analysis of <em>Anopheles gambiae</em>, <em>A. stephensi</em>, <em>Mycobacterium
                        tuberculosis</em> and <em>Candida glabrata</em>. Using
                        proteogenomic approach, we have corrected existing gene
                        models for 199 genes, identified 35 novel genes, and
                        assigned several novel translational start sites in the
                        genome of <em>A. gambiae</em>. </p>
                      <p>
                        <!--<a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('/projects/hprd.htm','hprd','width=600,height=700')"></a></p>-->
                    </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td width="31%" align="center"><img src="images/neuroproteomics.gif" style="padding: 4px; border: 1px solid #000000;" width="177" height="144" border="0"></td>
                  <td width="64%"><p><span class="header">Neuroproteomics</span></p>
                      <p align="justify">We are adopting proteomic, phosphoproteomic,
                        glycoproteomic approaches to identify molecular markers
                        for several neurological diseases such as chronic meningitis,
                        rabies, temporal lobe epilepsy and neurological stroke.&nbsp; </p>
                      <!--<p><a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('/projects/biobuilder.htm','biobuilder','scrollbars=yes,width=500,height=600')"></a></p>-->
                  </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><img src="images/mango_1.gif" style="padding: 4px; border: 1px solid #000000;" width="170" height="128" border="0"></td>
                  <td><p><span class="header">Plant proteomics </span>
                      <p align="justify"><em>Mangifera indica</em>, national
                        fruit of India, is one of important fruit crop. Although
                        extensive investigations have been carried out in raising
                        improved varieties of mango cultivars, molecular characterization
                        remains unexplored. Towards this end, we are carrying
                        out proteomic profiling of mango to provide a platform
                        for future investigations on molecular level to study
                        crop improvement, disease resistance and pest resistance.
                        This is particularly challenging because genome of mango
                        hasn&rsquo;t been sequenced yet. Therefore, we are depending
                        on homology based search and de novo sequencing to annotate
                        mass spectrometry derived data. </p>
                      <p align="justify">&nbsp;</p>
                      <p>
                        <!--<a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('/projects/genomebrowser.htm','genomebrowser','scrollbars=yes,width=500,height=600')"></a></p>-->
                    </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><img src="images/Toxicoproteomics_tr.gif" style="padding: 4px; border: 1px solid #000000;" width="170" height="128" border="0"></td>
                  <td><p><span class="header">Toxicoproteomics </span>
                      <p align="justify">We are using quantitative proteomics,
                        phosphoproteomics, glycoproteomics and cell line secretome
                        pipelines to study the response of human cells to toxic
                        pollutants. We are analyzing differential proteomics
                        of Hep G2 cells exposed to lead and arsenic with respect
                        to untreated controls. The differentially regulated proteins
                        may serve as candidate biomarker for diagnosing lead
                        or arsenic toxicity and for deciding treatment options.
                        Quantitative profiling of phosphoproteome in the cell
                        lines exposed to lead using high resolution mass spectrometry
                        may help us to identify signaling pathways activated
                        or inhibited upon lead toxicity. </p>
                      <p align="justify">&nbsp;</p>
                      <p>
                        <!--<a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('/projects/genomebrowser.htm','genomebrowser','scrollbars=yes,width=500,height=600')"></a></p>-->
                    </td>
                  <td>&nbsp;</td>
                </tr>
                <tr>
                  <td colspan="4">&nbsp;</td>
                </tr>
                <tr>
                  <td valign="top">&nbsp;</td>
                  <td align="center"><img src="images/ethnoproteomics_tr.gif" style="padding: 4px; border: 1px solid #000000;" width="170" height="128" border="0"></td>
                  <td><p><span class="header">Ethnoproteomics </span>
                      <p align="justify">India is rich in terms of cultural knowledge
                        associated with traditional medicine, mainly comprised
                        of usage of herbs. We are adopting the ethnobotanical
                        principles to test their effect on diseases in particular
                        or on human physiology in general using quantitative
                        mass spectrometry approaches. For example, curcumin,
                        a product of Curcuma longa, has been identified as one
                        of the major natural anticancer agents exerting anti-neoplastic
                        activity in various types of cancers tested. Using diverse
                        proteomic platforms we specifically identify proteins
                        that are differentially regulated in breast cancer upon
                        curcumin treatment. We are investigating differentially
                        induced phosphorylation of proteins in breast cancer
                        cells upon curcumin treatment. We can also identify molecules
                        and signaling pathways that are associated with anti-tumor
                        activity of curcumin by means of data generated by global
                        proteome and phosphoproteome profiles. It is possible
                        that a better understanding of the mechanistic details
                        of curcumin action in breast cancer will lead to design
                        of more targeted strategies to combat breast cancer. </p>
                      <p align="justify">&nbsp;</p>
                      <p>
                        <!--<a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('/projects/genomebrowser.htm','genomebrowser','scrollbars=yes,width=500,height=600')"></a></p>-->
                    </td>
                  <td>&nbsp;</td>
                </tr>
                <!-- <tr> 
                <td colspan="4">&nbsp;</td>
              </tr>
              <tr> 
                <td valign="top">&nbsp;</td>
                <td width="31%" align="center"><a href="http://plasmaproteomedatabase.org/" target="_blank"><img src="images/ppd.gif" border="1"></a></td>
                <td width="64%"><p><a href="http://plasmaproteomedatabase.org/" target="_blank" class="header">PLASMA PROTEOME DATABASE (PPD)</a></p>
                  <p>The Plasma Proteome Database, the first of its kind ensures a comprehensive resource for all human plasma proteins along with their isoforms. The database includes information pertaining to isoform specific expression, disease, localization, post translational modification and single nucleotide polymorphism. The information provided in this database is through manual annotation done by exhaustive literature research.</p>
                  <p><a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('http://plasmaproteomedatabase.org/','ppd','width=600,height=700')"></a></p></td>
                <td>&nbsp;</td>
              </tr>
			   <tr> 
                <td colspan="4">&nbsp;</td>
              </tr>
              <tr> 
                <td valign="top">&nbsp;</td>
                <td align="center"><img src="images/tagmapper_logo.jpg" width="177" height="52" border="0"></td>
                <td><p><a href="" class="header">TAGmapper</a></p>
                  <p>Is a comprehensive tool used to perform tag-to-gene mapping. 
                    The 10 base pair sequences of each of the SAGE tags, which 
                    are generated experimentally in the lab, are submitted to 
                    the tool as input. The output results in mapping the SAGE 
                    tags to their respective genes by performing an extensive 
                    search across the &#8216;dbEST&#8217; and &#8216;non redundant&#8217; 
                    database.</p>
                  <p><a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('/projects/tagmapper.htm','tagmapper','scrollbars=yes,width=500,height=600')"></a></p></td>
                <td>&nbsp;</td>
              </tr>
              <tr> 
                <td colspan="4">&nbsp;</td>
              </tr>
              <tr> 
                <td valign="top">&nbsp;</td>
                <td align="center"><a href="http://203.200.58.139/CG_Proj" target="_blank"><img src="images/cg_logo.jpg" width="177" height="53" border="0"></a></td>
                <td><p><a href="http://203.200.58.139/CG_Proj" target="_blank" class="header">ANALYSIS 
                    OF HUMAN 'X' CHROMOSOME</a></p>
                  <p>Involves a careful and comprehensive analysis of Human 'X' 
                    chromosome. Using comparative genomic approach, we have identified 
                    novel protein coding regions; we have performed an extensive 
                    pseudogene analysis of the 'X' chromosome and have documented 
                    alternative splicing events. In order to help the scientific 
                    community working on X-Linked Mental Retardation, the domain 
                    is linked to HPRD where the user can view the annotations 
                    of all the genes on 'X'.</p>
                  <p><a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('/projects/cg.htm','cg','scrollbars=yes,width=500,height=600')"></a></p></td>
                <td>&nbsp;</td>
              </tr>
             
              <tr> 
                <td colspan="4">&nbsp;</td>
              </tr>
              <tr> 
                <td valign="top">&nbsp;</td>
                <td align="center"><img src="images/ptpr_logo.jpg" width="175" height="40"></td>
                <td><p><a href="http://ptpr.ibioinformatics.org" class="header">ANALYSIS OF HUMAN 
                    PROTEIN TYROSINE PHOSPHATASES FOR ALTERNATIVE SPLICING</a></p>
                  <p> A thorough analysis of the protein tyrosine phosphatases 
                    encoded by the human genome using computational biology methods. 
                    Primary aim of the study is to identify novel tyrosine phosphatases 
                    and novel transcript variants for all the known protein tyrosine 
                    phosphatases. The findings are then experimentally validated 
                    by experts using techniques such as RT-PCRs and Northern blots.</p>
                  <p><a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('/projects/prf.htm','prf','scrollbars=yes,width=500,height=600')"></a></p></td>
                <td>&nbsp;</td>
              </tr>
              <tr> 
                <td colspan="4">&nbsp;</td>
              </tr>
              <tr> 
                <td valign="top">&nbsp;</td>
                <td align="center"><a href="http://www.oncomine.org" target="_blank"><img src="images/oncomine_logo.jpg" width="175" height="27" border="0"></a></td>
                <td><p><a href="http://www.oncomine.org/" target="_blank" class="header"> 
                    ONCOMINE</a></p>
                  <p>IOB, in collaboration with Chinnaiyan Lab, has created a microarray 
                    database whose goal is to curate publicly available cancer 
                    microarray studies and provide data mining tools to generate 
                    biologically relevant information in a user friendly manner. 
                    Links to various bioinformatics resources have been implemented 
                    including Unigene, Swissprot, Biocarta, HPRD, and KEGG, among 
                    others. IOB was involved in implementing the technical aspect 
                    for the database and Chinnaiyan Lab provided the normalized 
                    expression data. This database has been published in January/February 
                    issue of Neoplasia.</p>
                  <p><a href="#"><img src="images/more.gif" width="39" height="15" border="0" onClick="MM_openBrWindow('/projects/oncomine.htm','prf','scrollbars=yes,width=500,height=600')"></a></p></td>
                <td>&nbsp;</td>
              </tr>-->
                <tr>
                  <td colspan="4" align="right"><a href="#top">TOP</a></td>
                </tr>
              </table></td>
          </tr>
          <tr>
            <td colspan="3" align="left">&nbsp;</td>
          </tr>
        </table></td>
      </tr>
      <tr>
        <td colspan="3" align="left">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="3" align="left">&nbsp;</td>
      </tr>
    </table>
    
	
	</fieldset></td>
  </tr>
  
  
  </table>
  
  
  
  
  </td></tr> 
    </table>


<br>

<br>
</body>
</html>
