<html>
<head>
<link rel="shortcut icon" href="images/iob.ico" type="image/x-icon" />
<title>Institute of Bioinformatics</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<meta name="description" content="The Institute of Bioinformatics (IOB) established by Dr. Akhilesh Pandey, is a non-profit academic research organization located at International Tech Park in Bangalore, India. IOB is recognized by the Department of Scientific and Industrial Research, Government of India." /> 
<meta name="keywords" content="bioinformatics, india, bangalore, database, institute, proteomics, mass spectrometry, mass spectrometer, LTQ-Orbitrap, proteomics bangalore, india, genomics, protein reference database, oncomine, HPRD, akhilesh, akhilesh pandey, hprd, plasma proteome database, netpath, pathways, signaling pathways, pathbuilder, biobuilder, proteinpedia, human proteinpedia, SVM, RAPID, primary Immunodeficiency disease, cancer, biomarkers, biomarker, gstric cancer, pancreatic cancer, esophageal cancer, ESCC, X chromosome, XLMR, microarray, CGH array, neurological disorders, epilepsy, rabies, CNV, protein tyrosine phosphatases, gene expression arrays, tiling arrays, tissue microarrays, cell pellet arrays, breast cancer database, BCD, oncomine, phosphoproteomics, SILAC, iTRAQ, Glycoproteomics, proteogenomics, secretome, neuroproteomics, toxicoproteomics, plant proteomics, ethanoproteomics, serum proteomics, quantitative proteomics, HPLC, OFFGEL, nano-LC, ETD, Electron Transfer Dissociation, mycobacterium tuberculosis, research, proteomcis research, biomarker discovery, arsenic, Ph.D., manipal, kuvempu, pondicherry, amrita, rajiv gandhi university of health science, KIDWAI, NIMHANS, Johns Hopkins University, JHMI, RIKEN, RCAI, University of Toronto, IGIB, IICB, NIMR, MSKCC, sloan-kettering, University of Southern Denmark, HUPO, comparative proteomics, embryonic stem cells, anopheles, iob, comparative genomics, est">
<link rel="stylesheet" href="css/style.css" />
<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.5.2/jquery.min.js"></script>
<script type="text/javascript" src="jquery.aw-showcase.js"></script>
<script type="text/javascript">

$(document).ready(function()
{
	$("#showcase").awShowcase(
	{
	content_width:			540,
		content_height:			236,
		fit_to_parent:			false,
		auto:					true,
		interval:				3000,
		continuous:				false,
		loading:				true,
		tooltip_width:			200,
		tooltip_icon_width:		32,
		tooltip_icon_height:	32,
		tooltip_offsetx:		18,
		tooltip_offsety:		0,
		arrows:					true,
		buttons:				true,
		btn_numbers:			true,
		keybord_keys:			true,
		mousetrace:				false, /* Trace x and y coordinates for the mouse */
		pauseonover:			true,
		stoponclick:			false,
		transition:				'hslide', /* hslide/vslide/fade */
		transition_delay:		0,
		transition_speed:		500,
		show_caption:			'onload', /* onload/onhover/show */
		thumbnails:				false,
		thumbnails_position:	'outside-last', /* outside-last/outside-first/inside-last/inside-first */
		thumbnails_direction:	'vertical', /* vertical/horizontal */
		thumbnails_slidex:		1, /* 0 = auto / 1 = slide one thumbnail / 2 = slide two thumbnails / etc. */
		dynamic_height:			false, /* For dynamic height to work in webkit you need to set the width and height of images in the source. Usually works to only set the dimension of the first slide in the showcase. */
		speed_change:			true, /* Set to true to prevent users from swithing more then one slide at once. */
		viewline:				false, /* If set to true content_width, thumbnails, transition and dynamic_height will be disabled. As for dynamic height you need to set the width and height of images in the source. */
		custom_function:		null /* Define a custom function that runs on content change */
	});
});

</script>
<script language="JavaScript">
<!--

function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}
//-->
</script>

<link href="style.css" rel="stylesheet" type="text/css">
</head>
<body bgcolor="#E6E6E6" text="#000000" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0" onLoad="MM_preloadImages('images/collagiate/iob_roll.gif','images/buttons/csa_roll.gif','images/buttons/director_roll.gif','images/buttons/board_roll.gif','images/buttons/links_roll.gif','images/collagiate/team_roll.gif')">
<table width="801" border="0" align="center" cellpadding="0" cellspacing="0">

  <tr> 
    <td width="801"  bgcolor="#FFFFFF" align="center"><?php require("header.htm");?> &nbsp;</td>
  </tr> 
   <tr>
  <td>
  
  <table bgcolor="#FFFFFF" width="803">
  <tr><td width="129" valign="top"><?php include('menu.html');?> </td>
    <td width="662" valign="top" align="left"><fieldset><table width="100%">
	 <tr>
    <td colspan="3"   style="border-bottom:#0066FF medium groove; font:Geneva, Arial, Helvetica, sans-serif; font-size:14px" align="center"><strong>DATABASES</strong></td>
  </tr>
      <tr>
        <td colspan="2">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="2"><strong>Databases developed at IOB: </strong></td>
      </tr>
      <tr>
        <td colspan="2">&nbsp; </td>
      </tr>
      <tr>
        <td colspan="2"><strong> Defining the 'Proteome' </strong></td>
      </tr>
      <tr>
        <td colspan="2"><div align="justify">The term &quot;<em>Proteome</em>&quot;  defines the protein complement of a genome, has become the key research frontier in the post-genomic era. With the recent advancements in genomic and proteomic techniques, enormous amount of information available in literature about protein features such as protein-protein interactions, posttranslational modifications, sub-cellular localization, etc. Efforts such as fetching such information, storing them in a resource and disseminate them in easy to access web-based portals ease the task of reading literature for the scienctific researchers. To improve the efficiency of biological discoveries, Institute of Bioinformatics has developed various databases to identify and analyze protein products in a cell or tissue. These resources are freely available over the World Wide Web and can be easily accessed by the scientific community.</div>        </td>
      </tr>
      <tr><td colspan="2">&nbsp;</td></tr>
      <tr>
        <td colspan="2"><strong>Proteomic Resources</strong></td>
      </tr>
      <tr>
        <td>&nbsp;</td>
        <td>&nbsp;</td>
      </tr>
      <tr>
        <td align="center" valign="top">&#8226;</td>
        <td><div align="justify"><b>The Human Proteome Map</b> (HPM - <a href="http://humanproteomemap.org/" target="_blank">http://humanproteomemap.org</a>)
            portal is an interactive resource to the scientific community by integrating the massive peptide sequencing result from the draft map of the human proteome project. The project was based on LC-MS/MS by utilizing of high resolution and high accuracy Fourier transform mass spectrometry. All mass spectrometry data including precursors and HCD-derived fragments were acquired on the Orbitrap mass analyzers in the high-high mode. Currently, the HPM contains direct evidence of translation of a number of protein products derived from over 17,000 human genes covering >84% of the annotated protein-coding genes in humans based on >290,000 non-redundant peptide identifications of multiple organs/tissues and cell types from individuals with clinically defined healthy tissues. This includes 17 adult tissues, 6 primary hematopoietic cells and 7 fetal tissues. The HPM portal provides visualization of the expression pattern of a given gene across the sampled tissues using the label-free quantitative proteomic data set in a simple graphical view. It also provides the highly sampled peptides for given protein through interactive visualization. In addition, the portal provides selected reaction monitoring (SRM) information for all peptides identified.</div></td>
      </tr>
<!-- Template for Next Time 
	<tr><td width="4%">&nbsp;</td><td width="96%">&nbsp;</td></tr>
	<tr>
		<td align="center" valign="top">&#8226;</td>
		<td>
			<div align="justify">
				<strong>NetPath</strong> 
					<a href="http://www.netpath.org" target="_blank">http://www.netpath.org</a>
					) Description
			</div>
		</td>
	</tr>
-->
	<tr><td width="4%">&nbsp;</td><td width="96%">&nbsp;</td></tr>
	<tr>
		<td align="center" valign="top">&#8226;</td>
		<td>
			<div align="justify">
				<strong>Human Protein Reference Database</strong> 
					<a href="http://www.hprd.org" target="_blank"> (HPRD - http://www.hprd.org )</a>
					 is a centralized platform that stores and disseminates various information such as protein-protein interactions, post-translational modifications, tissue expression, sub-cellular localizations and enzyme-substrate relationships in normal condition. It stores these details for the currently known 30,047 human proteins. All information in this resource has been manually extracted from literature by expert biologists and reviewed by experienced researchers. HPRD also provides a tool called PhosphoMotif Finder (<a href="http://www.hprd.org/PhosphoMotif_finder" target="_blank">http://www.hprd.org/PhosphoMotif_finder</a>) to identify phosphorylation motifs which was developed using the manually curated motifs published in the literature.
			</div>
		</td>
	</tr>

	<tr><td width="4%">&nbsp;</td><td width="96%">&nbsp;</td></tr>
	<tr>
        <td align="center" valign="top">&#8226;</td>
        <td><div align="justify"><strong>NetPath</strong> (<a href="http://www.netpath.org" target="_blank">http://www.netpath.org</a>)
            and <strong>NetSlim</strong> (<a href="http://www.netpath.org/netslim/" target="_blank">http://www.netpath.org/netslim/</a>) NetPath is a resource to store manually curated signal transduction pathways in humans, that at present stores 37 signaling pathways. These pathways were manually curated by reading published literature and made publically available after publishing them in peer reviewed journals. NetSlim represents a subset of reactions depicted in NetPath and was developed by applying a set of stringent criteria to generate high-confidence signaling maps and also to enable easy visualization and interpretation of the pathways.
	</div></td>
      </tr>
	<tr><td width="4%">&nbsp;</td><td width="96%">&nbsp;</td></tr>
      <tr>
        <td align="center" valign="top">&#8226;</td>
        <td><div align="justify"><strong>Human Proteinpedia</strong> (<a href="http://www.humanproteinpedia.org/" target="_blank">http://www.humanproteinpedia.org/</a>) This resource is developed as a community portal for sharing and integrating human protein data derived from various high throughput as well as low throughput experiments. It allows the users to visualize the results of these experiments related to a protein of interest in a single page. This resource allows research labs around the world to contribute and maintain protein annotations derived from various platforms like mass spectrometry, immunochemistry and fluorescence based experiments.</div></td>
      </tr>

	<tr><td width="4%">&nbsp;</td><td width="96%">&nbsp;</td></tr>

      <tr>
        <td align="center" valign="top">&#8226;</td>
        <td><div align="justify"><strong>Plasma Proteome Database</strong> (<a href="http://www.plasmaproteomedatabase.org/" target="_blank">http://www.plasmaproteomedatabase.org/</a>)
            Plasma Proteome Database (PPD) was developed as a part of Human Proteome Organization's (HUPO) initial effort to characterize human plasma proteome. This is one of the largest resources on proteins reported in plasma and serum. PPD hosts qualitative and quantitative information on proteins (including those from MRM-based assays) reported in plasma and serum and hence serves as reference platform for biomarker discovery. In addition, proteins reported in extracellular vesicle isolated from plasma are also available. PPD query interface is designed to facilitate individual and batch queries using gene symbols, PubMed identifiers, RefSeq accessions, UniProt IDs or peptide sequences. Each of the protein is linked to external resources such as Entrez Gene, Human Protein Reference Database (HPRD), UniProt and NetPath for more details. It also stores the concentration of a given protein in blood, plasma or serum which could be used for diagnostics in several diseases.
	</div></td>
      </tr>

	<tr><td width="4%">&nbsp;</td><td width="96%">&nbsp;</td></tr>

      <tr>
        <td align="center" valign="top">&#8226;</td>
        <td><div align="justify"><strong>The Pancreatic Cancer Database </strong>(<a href="http://pancreaticcancerdatabase.org/" target="_blank">http://pancreaticcancerdatabase.org/</a>) is a resource of experimentally demonstrated molecular alterations associated with pancreatic cancer in cancer tissues or cancer cell lines. It currently contains information pertaining to changes at the mRNA, protein and miRNA levels. The data can be queried or browsed at mRNA, protein or miRNA levels or based on specific cancer subtypes.</div></td>
      </tr>

	<tr><td width="4%">&nbsp;</td><td width="96%">&nbsp;</td></tr>
      <tr>
        <td align="center" valign="top">&#8226;</td>
        <td><div align="justify"><strong>Resource of Asian Primary Immunodeficiency
              Diseases</strong> (<a href="http://rapid.rcai.riken.jp/RAPID" target="_blank">http://rapid.rcai.riken.jp/RAPID</a>)
            RAPID is a web-based compendium of molecular alterations in primary immunodeficiency diseases. It provides detailed information about genes and proteins that are altered such as protein-protein interactions, microarray gene expression profiles and deleterious and novel mutations associated with them. This was joint effor by IOB and RIKEN, Japan.</div></td>
      </tr>

	<tr><td width="4%">&nbsp;</td><td width="96%">&nbsp;</td></tr>
      <tr align="left">
        <td colspan="2" valign="top">
          <strong>Other Resources:          </strong></td>
      </tr>

	<tr><td width="4%">&nbsp;</td><td width="96%">&nbsp;</td></tr>
	<tr>
		<td align="center" valign="top">&#8226;</td>
		<td>
			<div align="justify">
				<strong>Stable isotope labeling by amino acids in cell culture </strong> (SILAC -    
					<a href="http://silac.org/index_html" target="_blank">http://silac.org/index_html</a>
					) SILAC is an <i>in vivo</i> labelling method for mass spectrometry-based quantitative proteomics. It is a simple, reproducible and versatile technique with numerous applications. Since its inception, SILAC has evolved with several variations in media, reagents, and methods in analysing data. Owing to the large volume of information out there and its complexity, we developed a web resource to facilitate easy access to the available information pertaining to the methodologies involved and applications of this technique.
			</div>
		</td>
	</tr>

	<tr><td width="4%">&nbsp;</td><td width="96%">&nbsp;</td></tr>

      <tr>
        <td align="center" valign="top">&#8226;</td>
        <td><div align="justify"><strong>TBnet</strong> (<a href="http://tbnetindia.ibioinformatics.org/" target="_blank">http://tbnetindia.ibioinformatics.org/</a>) This resource was developed as part of an initiative by Department of Biotechnology, Government of India to show Indian contributions towards the research to address various issues related to tuberculosis. The portal describes theinformation about genomics and proteomics advances, genetic diversity, clinical therapeutics and host-pathogen interactions related to tuberculosis.</div></td>
      </tr>
	<tr><td width="4%">&nbsp;</td><td width="96%">&nbsp;</td></tr>
      <tr>
        <td align="center" valign="top">&#8226;</td>
        <td><div align="justify"><strong>India Cancer Research Database</strong> <a href="http://www.incredb.org/" target="_blank">(http://www.incredb.org/)</a> This database provides details of scientists and physicians involved in cancer research in India along with the information about their areas of expertise, research publications and funded grants. The main goal of this database is to foster collaborations among researchers and to provide a snapshot of ongoing research initiatives and activities in India. <br>
        </div></td>
      </tr>
      <tr>
        <td>&nbsp;</td>
        <td align="right">&nbsp;</td>
      </tr>
      <tr>
        <td>&nbsp;</td>
        <td align="right"><a href="more_resources.php"><img src="images/more.gif" width="39" height="15" border="0"></a></td>
      </tr>
    </table>
    </fieldset></td>
  </tr>
  
  
  </table>
  
  
  
  
  </td></tr> 
    </table>


<br>

<br>
</body>
</html>
